packages/amplify_core/ios/Classes/SwiftCore.swift

```diff
@@ -33,27 +33,26 @@ public class SwiftCore: NSObject, FlutterPlugin {
   public func handle(_ call: FlutterMethodCall, result: @escaping FlutterResult) {
     switch call.method {
       case "configure":
-        do {
-          let arguments = call.arguments as! Dictionary<String, AnyObject>
-          let version = arguments["version"] as! String
-          let configuration = arguments["configuration"] as! String
-          let amplifyConfiguration = try JSONDecoder().decode(AmplifyConfiguration.self, from: configuration.data(using: .utf8)!)
-          onConfigure(result: result, version: version, amplifyConfiguration: amplifyConfiguration)
-            
-        } catch {
-            print("Failed to configure Amplify \(error)")
-            result(FlutterError(code: "AmplifyException",
-                                message: "Failed to Configure Amplify",
-                                details: error.localizedDescription));
-        }
+        let arguments = call.arguments as! Dictionary<String, AnyObject>
```

Comment:
nit: These values never fail? ie call.arguments always contains Dictionary? I am concerned with the force casting. 
If this is intended, I guess adding some comments would make it clear.

Reply:
yeah we need to put guards here. This is an earlier category that we didn't add much error handling. We will be addressing this in future PRs.